javascript/javascript1/week3/homework-js1-w3.js

```diff
@@ -0,0 +1,249 @@
+//**********************************************/
+// WARM UP
+//**********************************************/
+
+//--------------------------
+// Item array removal
+//--------------------------
+const names = [
+  "Peter",
+  "Ahmad",
+  "Yana",
+  "kristina",
+  "Rasmus",
+  "Samuel",
+  "katrine",
+  "Tala",
+];
+const nameToRemove = "Ahmad";
+
+for (let i = 0; i < names.length; i++) {
+  if (names[i] === nameToRemove) {
+    names.splice(i, 1); // cannot use slice() here because slice doesn't modify the original array
+  }
+}
```

Comment:
There is a nifty array method for finding the exact index of a matching element:
```
names.indexOf(nameToRemove)
```